docs(pie-docs): DSW-2211 adding overview of web components (#1589)
justeattakeaway/pie: apps/pie-docs/src/engineers/getting-started/overview.md

---

> ## PIE component systems

Reviewer: Can we come up with a clearer title? This one doesn't give me many hints about what the content will be talking about

Author: Would it work to say `PIE component libraries` ?
Or if we include this content as part of the introduction, we can divided like so:
(If this does not work, let me know a suggestion :) )
<img width="810" alt="Screenshot 2024-07-17 at 11 59 32" src="https://github.com/user-attachments/assets/429ecaad-ba0d-430e-9529-7a2f124b1133">

Author: Decided to remove the smaller headings but left the new content as part of the Introduction section.

> We develop and maintain different projects that incorporate the PIE Design Language. If you would like to know which one to use, please follow [this guide](/engineers/getting-started/structure/#what-library-to-use) to understand what is the best library for you, based on your tech stack.

Reviewer: ```suggestion
We develop and maintain several component libraries that implement the PIE design language. Please follow [this guide](/engineers/getting-started/structure/#what-library-to-use) to find out which library to use based on your tech stack.
```

For a more detailed look at the **component systems** the PIE team maintain, the PIE monorepo and how it’s structured, see the [structure section of our engineering docs](/engineers/getting-started/structure).

> For a list of what components are supported in our different libraries please refer to the [Components Status page](/components/component-status/).

Reviewer: ```suggestion
For a list of which components are supported in each of our libraries, please refer to the [Component Status page](/components/component-status/).
```
